src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -70,6 +70,8 @@
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
     <TSqlScript Include="Features\Schema\Sql\Types\*.sql" />
     <None Remove="Features\Schema\Migrations\10.diff.sql" />
+    <None Remove="Features\Schema\Sql\Sprocs\AddWorkitemV10.sql" />
```

Comment:
Please remove this 4 lines, we dont need . It is included as part of line 65

Reply:
Done